### PR TITLE
HDDS-11480. Refactor OM volume response tests

### PR DESCRIPTION
**Pull request by harryteng9527**
## What changes were proposed in this pull request?

This PR extracts the bootstrap methods (`setup()`, `tearDown()`) used in the following OM volume response tests to reduce redundant code by allowing the tests to inherit from a superclass:

- TestOMVolumeCreateResponse
- TestOMVolumeDeleteResponse
- TestOMVolumeSetOwnerResponse
- TestOMVolumeSetQuotaResponse

## What is the link to the Apache JIRA

[HDDS-11480. Refactor OM volume response tests](https://issues.apache.org/jira/browse/HDDS-11480)

## How was this patch tested?

- Pass all unit test

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/TestOMVolumeResponse.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.apache.hadoop.ozone.om.response.volume;
```
Comment (ivandika3) [inline hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/TestOMVolumeResponse.java:1]: Please add the license header.

**Review by ivandika3 — COMMENTED** [review]
@harryteng9527 Thanks for picking this up. Overall looks good. Could you help resolve the checkstyle and rat (license) failure first?

**Review by sarvekshayr — COMMENTED** [review]
Thanks @harryteng9527 for this patch.
LGTM, waiting for green CI.

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/TestOMVolumeResponse.java (hunk 1 above)
Comment (harryteng9527) [reply hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/TestOMVolumeResponse.java:19]: Thank you for your review. I have added the license and resolved the checkstyle issues.

**Review by ivandika3 — APPROVED** [review]
@harryteng9527 Thanks for the update. LGTM +1.

**Comment by ivandika3** [comment]
Thanks @harryteng9527 for the patch and @sarvekshayr for the review.

**Pull request merged**